src/nccl_ofi_rdma.c

```diff
@@ -2214,7 +2384,10 @@ static int test(nccl_net_ofi_req_t *base_req, int *done, int *size)
 		/* Mark as done */
 		*done = 1;
 
-		if (req->type != NCCL_OFI_RDMA_FLUSH) {
+		if (req->type != NCCL_OFI_RDMA_FLUSH &&
+		    req->type != NCCL_OFI_RDMA_WRITE &&
+		    req->type != NCCL_OFI_RDMA_WRITE_DATA &&
+		    req->type != NCCL_OFI_RDMA_READ) {
```

Comment:
Nit: this can be simply `if (req->type == NCCL_OFI_RDMA_SEND || req->type == NCCL_OFI_RDMA_RECV)`

Reply:
Done